packages/shared/lib/models/Sync.ts

```diff
@@ -158,17 +157,20 @@ export interface RunScriptOptions {
     syncConfig?: SyncConfig;
     syncName: string;
     syncId: string;
-    activityLogId: number | undefined;
+    activityLogId?: number | undefined;
     nangoProps: NangoProps;
-    integrationData: Pick<NangoIntegrationData, 'fileLocation'>;
-    environmentId: number;
     writeToDb: boolean;
     isInvokedImmediately: boolean;
     isWebhook: boolean;
     optionalLoadLocation?: string | undefined;
     input?: object | undefined;
 }
-export interface IntegrationServiceInterface {
+
```

Comment:
this below is temporary. It will change drastically when I refactor and move `run.service`